parts/k8s/cloud-init/jumpboxcustomdata.yml

```diff
@@ -11,12 +11,16 @@ write_files:
 
 - path: "/home/{{WrapAsParameter "jumpboxUsername"}}/.kube/config"
   permissions: "0644"
-  owner: "root"
+  owner: "{{WrapAsParameter "jumpboxUsername"}}"
   content: |
 {{WrapAsVariable "kubeconfig"}}
 
 runcmd:
 - . /opt/azure/containers/provision_source.sh
 - retrycmd_if_failure 10 5 10 curl -LO https://storage.googleapis.com/kubernetes-release/release/v{{.OrchestratorProfile.OrchestratorVersion}}/bin/linux/amd64/kubectl
 - chmod +x ./kubectl
-- sudo mv ./kubectl /usr/local/bin/kubectl
\ No newline at end of file
+- sudo mv ./kubectl /usr/local/bin/kubectl
+- chown -R "{{WrapAsParameter "jumpboxUsername"}}" "/home/{{WrapAsParameter "jumpboxUsername"}}"
+- chgrp -R "{{WrapAsParameter "jumpboxUsername"}}" "/home/{{WrapAsParameter "jumpboxUsername"}}"
+- chown -R root "/home/{{WrapAsParameter "jumpboxUsername"}}/.kube"

```

Comment:
why do we want `~/.kube` to be owned by root?

Reply:
@CecileRobertMichon see my reply to @mboersma review. It's a defensive minimal change and actually doesn't impose any restriction as the config can be read. If the user want's to change he can, because the default user usually has sudo rights.